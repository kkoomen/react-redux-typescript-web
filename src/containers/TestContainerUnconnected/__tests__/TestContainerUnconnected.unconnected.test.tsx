import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { TestContainerUnconnected } from '../';

describe('TestContainerUnconnected container', () => {
  it('should match its snapshot by default', () => {
    const element = shallow(<TestContainerUnconnected />);
    expect(toJson(element)).toMatchSnapshot();
  });
});
