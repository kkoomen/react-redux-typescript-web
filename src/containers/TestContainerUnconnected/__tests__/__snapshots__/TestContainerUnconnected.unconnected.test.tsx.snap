// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestContainerUnconnected container should match its snapshot by default 1`] = `
<div
  className="TestContainerUnconnected"
/>
`;
